webpack.config.js

```diff
@@ -61,5 +61,6 @@ module.exports = {
   devServer: {
     contentBase: path.join(__dirname, "src"),
     historyApiFallback: true,
+    https: true,
```

Comment:
Won't this affect the deployment of the application?

Reply:
In order, for Facebook to work without errors. Https should be set to true